### DISCUSSION TITLE
希望能够翻译长文时，自定义每次翻译的段落长度。

### DISCUSSION THREAD

**Discussion by luluzi99** [immersive-translate/immersive-translate#2164] [Ideas]
### 新功能 | New Feature

每次用插件看英文长文的时候，总是将一大段英文整体翻译成中文显示在下面，不利于和中文对比，希望能够出个自定义每次翻译段落长度的功能，现在每次最多翻译大概150个单词左右，希望能自定义50 100 150 三个挡位的每次英文翻译的长度。

**Comment by theowenyoung** [comment]
可以在【进阶设置】设置长短落自动换行
